walk-in-interview/src/main/webapp/update-job/script.js

```diff
@@ -405,7 +443,7 @@ cancelButton.addEventListener('click', (_) => {
  * since getJobId is not implemented yet.
  */
 function getJobIdForSeleniumTest() {
```

Comment:
I think you should just use getJobId function in the selenium tests rather than having this function here now and then deleting it later

Reply:
i think it will be better to separate test-used code and normal code